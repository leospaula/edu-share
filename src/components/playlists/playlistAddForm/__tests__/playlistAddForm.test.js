import React from 'react'

it ('renders without crashing', () => {

})

it ('matches the snapshot', () => {

})

it ('redirects to the /bookshelves route after submission if brought up from that route', ()=> {

})

it ('redirects to the /bookshelves route after submission if brought up from that route', ()=> {

})

it ('triggers addPlaylist function on submission', () => {

})

it ('triggers addPlaylist with the correct arguments on submission', () => {
  
})