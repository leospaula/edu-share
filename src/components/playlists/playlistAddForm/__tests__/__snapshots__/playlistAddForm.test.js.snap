// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <label>
      Playlist Name: 
    </label>
    <input
      className="form-control"
      name="name"
      onChange={[Function]}
      type="text"
      value=""
    />
    <br />
  </div>
  <input
    className="btn btn-primary tertiary-background"
    data-testid="add-playlist-submit"
    type="submit"
    value="add"
  />
</form>
`;
